Abandon the WITH_LANG=ALL building of all languages until I know why
Index: source/templates/wizard/letter/lang/makefile.mk
===================================================================
RCS file: /cvs/installation/extras/source/templates/wizard/letter/lang/makefile.mk,v
retrieving revision 1.10
diff -u -r1.10 makefile.mk
--- extras/source/templates/wizard/letter/lang/makefile.mk	3 Nov 2005 17:08:44 -0000	1.10
+++ extras/source/templates/wizard/letter/lang/makefile.mk	12 Jan 2006 10:16:59 -0000
@@ -34,7 +34,7 @@
 PRJ=..$/..$/..$/..$/..
 
 # special handling here to fit SO requirements
-WITH_LANG!:=ALL
+# WITH_LANG!:=ALL
 
 PRJNAME=extras
 TARGET=templateswizardletter
@@ -45,17 +45,17 @@
 
 
 ZIPFLAGS                =
-ZIP1TARGET      = $(TEMPLATE_WIZARD_LETTER_TARGET)
+# ZIP1TARGET      = $(TEMPLATE_WIZARD_LETTER_TARGET)
 
 # */*.ott is here on purpose. It works in all build environments.
 # Do not use $/ here.
-ZIP1LIST	= */*.ott
+# ZIP1LIST	= $(LANGDIR)/*.ott
 
-ZIP2TARGET      = $(TEMPLATE_WIZARD_LETTER_ONELANG_TARGET)
+ZIP1TARGET      = $(TEMPLATE_WIZARD_LETTER_ONELANG_TARGET)
 
 # */*.ott is here on purpose. It works in all build environments.
 # Do not use $/ here.
-ZIP2LIST	= $(LANGDIR)/*.ott
+ZIP1LIST	= $(LANGDIR)/*.ott
 
 
 
